wolfcrypt/src/pkcs12.c

```diff
@@ -1060,6 +1144,130 @@ static WARN_UNUSED_RESULT int freeDecCertList(WC_DerCertList** list,
     return 0;
 }
 
+#ifdef ASN_BER_TO_DER
+/* append data to encrypted content cache in PKCS12 structure
+ * return buffer on success, NULL on error */
+static byte* PKCS12_ConcatonateContent(byte* mergedData, word32* mergedSz,
+        byte* in, word32 inSz)
+{
+    byte* oldContent;
+    word32 oldContentSz;
+
+    if (mergedData == NULL || in == NULL)
+        return NULL;
+
+    /* save pointer to old cache */
+    oldContent = mergedData;
+    oldContentSz = *mergedSz;
+
+    /* re-allocate new buffer to fit appended data */
+    mergedData = (byte*)XMALLOC(oldContentSz + inSz, NULL, DYNAMIC_TYPE_PKCS);
+
+    if (oldContent != NULL) {
+        XMEMCPY(mergedData, oldContent, oldContentSz);
+    }
+    XMEMCPY(mergedData + oldContentSz, in, inSz);
+    *mergedSz += inSz;
+
+    XFREE(oldContent, NULL, DYNAMIC_TYPE_PKCS);
+
+    return mergedData;
+}
+
+
+
+static int PKCS12_HandleIndefinite(WC_PKCS12* pkcs12, byte* data, word32 dataSz,
```

Comment:
Don't be lazy with naming!
What part of the PKCS#12 is this handling?
It is not just indefinite that it handles here either!

Reply:
Fair point. Done.